### PR TITLE
show-balance-fraction

### PR DESCRIPTION
**Pull request by jimscarver**
show rev balance with decimal point

### CODE REVIEW

File: src/actions/checkBalance.rho [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
+      balanceCh
+      in {
+      lookup!(`rho:rchain:revVault`, *RevVaultCh) |
+      for (@(_, RevVault) <- RevVaultCh) {
+         @RevVault!("findOrCreate", myGovRevAddr, *vaultCh) |
+         for (@(true, vault) <- vaultCh) {
+            @vault!("balance", *balanceCh) |
+            for (@balance <- balanceCh) {
+            return!(["#define", "$myBalance", balance])|
+            return!("${rev}.${fraction}" %% {
```
Comment (Bill-Keuntje) [inline src/actions/checkBalance.rho:17]: @jimscarver I've never seen this syntax before. Can you educate me?

Comment (jimscarver) [reply src/actions/checkBalance.rho:17]: string interpolation from the cheatsheet 
image
https://rchain-community.github.io/tutorials/cheat-sheet/

**Review by Bill-Keuntje — APPROVED** [review]

File: src/actions/checkBalance.rho (hunk 1 above)
Comment (Bill-Keuntje) [reply src/actions/checkBalance.rho:17]: Nice.

Comment (w2vy) [reply src/actions/checkBalance.rho:17]: This syntax seems to cause issued with pyrchain
`Traceback (most recent call last):
  File "/home/tom/RChain/rgov/testing/unit_test/./test_balance.py", line 10, in <module>
    balance = rgov.checkBalance(new1.get_public_key().get_rev_address())
  File "/home/tom/RChain/rgov/pyrgov/rgov.py", line 137, in checkBalance
    result = self.client.exploratory_deploy(contract, block_hash)
  File "/home/tom/RChain/pyrchain/rchain/client.py", line 119, in exploratory_deploy
    self._check_response(response)
  File "/home/tom/RChain/pyrchain/rchain/client.py", line 91, in _check_response
    raise RClientException('\n'.join(response.error.messages))
rchain.client.RClientException: syntax error(): / at 27:4-27:5
`
From rnode.log

`coop.rchain.rholang.interpreter.errors$SyntaxError: syntax error(): / at 27:4-27:5
	at coop.rchain.rholang.interpreter.compiler.ErrorHandlingParser.unrecovered_syntax_error(ParBuilder.scala:138)
	at java_cup.runtime.lr_parser.parse(lr_parser.java:731)
	at coop.rchain.rholang.syntax.rholang_mercury.parser.pProc(parser.java:1226)
	at coop.rchain.rholang.interpreter.compiler.ParBuilder$$anon$1.$anonfun$buildAST$3(ParBuilder.scala:48)
	at cats.effect.Sync.$anonfun$delay$1(Sync.scala:54)
	at cats.effect.Sync$KleisliSync.$anonfun$suspend$6(Sync.scala:314)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
	at runToFuture @ coop.rchain.catscontrib.TaskContrib$TaskOps$.unsafeRunSync$extension(taskOps.scala:27)
`

Comment (jimscarver) [reply src/actions/checkBalance.rho:17]: the last line comment had no return on the end of the line. I fixed it in master.

Comment (Bill-Keuntje) [reply src/actions/checkBalance.rho:17]: Yep, rholang files should have a final ```\n``` at the end of the file

**Pull request merged**